packages/fxa-auth-server/lib/payments/stripe.js

```diff
@@ -95,7 +95,9 @@ class StripeHelper {
     this.plansCacheIsEnabled = this.cacheTtlSeconds && redis;
 
     this.stripe = new stripe(config.subscriptions.stripeApiKey, {
+      apiVersion: '2019-12-03',
       maxNetworkRetries: 3,
+      typescript: undefined,
```

Comment:
I cannot find what values are allowed here on their README.  I guess `undefined` is essentially the same as `false`?

Reply:
Yes, though oddly the way they setup the typing, if you include this object at all, then it *must* include the `typescript` flag... which can only be true. This is just an artifact of the weird typing they did.

Reply:
I filed an issue with the stripe lib to not require this value.